src/site/content/en/blog/building-a-sidenav-component/index.md

```diff
@@ -0,0 +1,333 @@
+---
+layout: post
+title: Building a sidenav component
+subhead: A foundational overview of how to build a responsive slide out sidenav
+authors:
+  - adamargyle
+description: A foundational overview of how to build a responsive slide out sidenav
+date: 2021-01-16
+hero: hero.png
```

Comment:
Can you run this hero through an optimizer and try to get its size down a bit? It doesn't have to be an aggressive optimization. Just trying to keep the size of the repo down.